### PR TITLE
Update to GBBPEx v005

### PR DESCRIPTION
**Pull request by zhanglikate**
# Description

# Type of change

- Bug fix (fixes something broken)
- New feature (adds functionality)
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? YES/NO
- Does this change require a documentation update? YES/NO

# How has this been tested?

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: jobs/rocoto/prepchem.sh [hunk 1]
```diff
@@ -109,12 +109,18 @@ for n in $(seq 1 6); do
       else
 
         if [ ${SYEAR} -eq 2016 -o ${emiss_date1} -ge 20230115 ];  then           ## JKH - change date
-          if [ ${emiss_date1} -ge 20230509 ];  then                              ## JKH - change date
+          if [ [ ${emiss_date1} -ge 20230509 ] && [ ${emiss_date1} -le 20240501 ] ];  then                              ## JKH - change date
```
Comment (jderrico-noaa) [inline jobs/rocoto/prepchem.sh]:
```suggestion
          if [[ ${emiss_date1} -ge 20230509 && ${emiss_date1} -le 20240501 ]]; then                              ## JKH - change date
```

**Review by jderrico-noaa — COMMENTED** [review]
Shellcheck flagged a syntax error with nested brackets for (potential) misuse of logical operators, can you let me know if this suggestion works?

File: jobs/rocoto/prepchem.sh (hunk 1 above)
Comment (zhanglikate) [reply jobs/rocoto/prepchem.sh]: I have updated. How can I do now?

**Review by jderrico-noaa — APPROVED** [review]

File: jobs/rocoto/prepchem.sh [hunk 2]
```diff
@@ -109,12 +109,18 @@
...
 
         if [ ${SYEAR} -eq 2016 -o ${emiss_date1} -ge 20230115 ];  then           ## JKH - change date
-          if [ ${emiss_date1} -ge 20230509 ];  then                              ## JKH - change date
+	   if [[ "${emiss_date1} -ge 20230509"  &&  "${emiss_date1} -le 20240501" ]]; 
             BC=GBBEPxemis-BC-${CASE}GT${n}_v4r0_${emiss_date1}.bin
             OC=GBBEPxemis-OC-${CASE}GT${n}_v4r0_${emiss_date1}.bin
             SO2=GBBEPxemis-SO2-${CASE}GT${n}_v4r0_${emiss_date1}.bin
             FRP=GBBEPxFRP-MeanFRP-${CASE}GT${n}_v4r0_${emiss_date1}.bin
             PM25=GBBEPxemis-PM25-${CASE}GT${n}_v4r0_${emiss_date1}.bin
+	  elif [[ "${emiss_date1} -ge 20240502" ]]; then
```
[the remark this answers is not in the document]
Comment (jderrico-noaa) [reply jobs/rocoto/prepchem.sh]:
```suggestion
	  elif [[ ${emiss_date1} -ge 20240502 ]]; then
```

File: jobs/rocoto/prepchem.sh [hunk 3]
```diff
@@ -109,12 +109,18 @@
       else
 
         if [ ${SYEAR} -eq 2016 -o ${emiss_date1} -ge 20230115 ];  then           ## JKH - change date
-          if [ ${emiss_date1} -ge 20230509 ];  then                              ## JKH - change date
+	   if [[ "${emiss_date1} -ge 20230509"  &&  "${emiss_date1} -le 20240501" ]];
```
[the remark this answers is not in the document]
Comment (jderrico-noaa) [reply jobs/rocoto/prepchem.sh]:
```suggestion
	   if [[ ${emiss_date1} -ge 20230509 && ${emiss_date1} -le 20240501 ]]; then 
```

**Review by jderrico-noaa — APPROVED** [review]

**Comment by jderrico-noaa** [comment]
Shell check was successful, so I'm merging this PR

**Comment by zhanglikate** [comment]
> Shell check was successful, so I'm merging this PR


No, it is has issue. I fixed one more time.

> Shell check was successful, so I'm merging this PR

**Comment by zhanglikate** [comment]
@jderrico-noaa I think I need to submit one more PR, the previous did not work.

**Pull request merged**